packages/docs-theme/src/styles/_layout.scss

```diff
@@ -34,10 +34,10 @@ Page layout elements
 */
 
 .docs-root {
-  background-color: $content-background-color;
+  background-color: $pt-app-background-color;
 
   &.#{$ns}-dark {
-    background-color: $dark-content-background-color;
+    background-color: $pt-dark-app-background-color;
```

Comment:
actually, strike that last comment -- the change to docs-theme is fine. the color aliases being used were inconsistent before; we were using "app background color" for light theme but something else for dark theme, and this change unifies them to be symmetrical. so it's a welcome fix for the docs-theme package and it doesn't darken things _so much_ that we need all the adjustments from #6711